lib/api.js

```diff
@@ -4,9 +4,9 @@ var url = require('url');
 var request = require('request');
 var Q = require('q');
 var xml2js = require('xml2js');
+var credentials = require('plex-api-credentials');
```

Comment:
Might want to move this inside of _credentialsAuthenticator() ?

It's not common practice to put a module require() inside a method, but otherwise we'll be loading in the full module (small as it is) even in cases where it won't ever be used.